ice/recipes/synthesize.py

```diff
@@ -4,6 +4,8 @@
 from ice.recipes.abstract_qa import Abstract
 from ice.recipes.abstract_qa import DEFAULT_ABSTRACTS
 
+import json
```

Comment:
In general the synthesize recipes are mediocre-quality code

I think that's fine

but it does mean that we may not want to merge them into the main ICE repo. Should we put them in an iceplus repo?